HTML:CSS learning project/style.css

```diff
@@ -7,25 +7,29 @@ header{
   top: 0;
 }
 
-footer {
+.main-footer {
   color:white;
   width: 100%;
   position: fixed;        
   background-color:black;
   padding: 10px 0;
   bottom: 0;
 }
-nav a{
+
+.main-nav a{
```

Comment:
use spacing before curly bracket just to keep it clean. What's important that if you have prettier installed on your editor then it does it for you automatically to save headaches from formatting.